### PR TITLE
What is anti-maware article

### PR DESCRIPTION
**Pull request by Slyracoon23**
Hi,

I know it has been a while since I published anything but I am back with articles! The pictures are probably not allowed since most are copyrighted but they can be taken away. 

Happy to hear any feedback

### CODE REVIEW

**Comment by hectorkambow** [comment]
Good afternoon/morning @jstangroome Jason 👋  - if you could please give this security article a tech review whenever you have a moment. 🚀  Thank you

**Comment by hectorkambow** [comment]
> Hi,
> 
> I know it has been a while since I published anything but I am back with articles! The pictures are probably not allowed since most are copyrighted but they can be taken away.
> 
> Happy to hear any feedback

👋 - Good afternoon Earl @Slyracoon23 - great to have you back - just started a preliminary review on your article - the images (in the articles) are completely fine - as long as we add the URL from where we got them from below them *cited* - not a problem. 
 
If we could just add those URLs - 🚀  Thank you 

The article is being pushed for a tech review now - pending and edits there - the article should be ready to publish.

File: articles/what-is-anti-virus-software/index.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+===============
+
+![bandit_arsenal](/articles/what-is-anti-virus-software/bandit_arsenal.jpg)
+
+Let’s now look at some of the major forms of malware.
+
+Viruses
+-------
+
+Computer viruses can run on any operating system be it Windows or Mac ( Mac’s are not immune ;) ). Computer viruses are actually, contrary to common belief, a type of malware. Viruses are defined as follows:
```
Comment (jstangroome) [inline articles/what-is-anti-virus-software/index.md:75]: If you like you can reference this recent article which found Mac has become more susceptible than Windows:
https://blog.malwarebytes.com/reports/2020/02/malwarebytes-labs-releases-2020-state-of-malware-report/amp/

Comment (Slyracoon23) [reply articles/what-is-anti-virus-software/index.md:76]: I added the link to the Malwarebytes
Thanks

Comment (Slyracoon23) [reply articles/what-is-anti-virus-software/index.md]: added link to malwarebytes

**Pull request merged**